src/shogun/multiclass/LaRank.h

```diff
@@ -485,6 +497,9 @@ namespace shogun
 
 			/// progess output
 			int32_t step;
+
+			/// max iterations
```

Comment:
this comment is useless -- it is just the variable name :)
I know it is quite self explaining here, but something like
"Maximum number of iterations before training is stopped"
is much more polished if users/developers read the doxygen docs